install.sh

```diff
@@ -109,6 +109,11 @@ install_nvm_from_git() {
       command git --git-dir="$INSTALL_DIR"/.git --work-tree="$INSTALL_DIR" branch -D master >/dev/null 2>&1
     fi
   fi
+
+  echo "=> Compressing and cleaning up git repository"
+  if ! command git --git-dir="$INSTALL_DIR"/.git --work-tree="$INSTALL_DIR" gc --aggressive --prune=now ; then
```

Comment:
Thanks for reminding.
git v1.5.5 does not support `--prune` with value, the interesting thing is the `git gc --aggressive` return `Nothing new to pack.`, so give an upgrade request may be a better workaround.